test_radio_comms.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python3
```

Comment:
What's the intended difference between test_radio_comms.py and winlink_canary.py?

Reply:
Its just a diagnostic program.  Let me make that clear in a comment at the top. This program is intended to stress tait.py and the radio's serial control to ensure that we don't have flaky control channel (which is sort of what we thought we we seeing).  The program helped me prove that at least in the absence of RF feedback the system was stable.